docs/_edot-sdks/java/setup/index.md

```diff
@@ -7,7 +7,52 @@ parent: EDOT Java
 
 # Setting up the EDOT Java Agent
 
-TODO:
-- where to download
-- explicit description of basic setup here (even if it overlaps with upstream docs)
-- link to upstream docs for more advanced setup use cases
\ No newline at end of file
+## Download
+
+Latest release: [![Maven Central](https://img.shields.io/maven-central/v/co.elastic.otel/elastic-otel-javaagent?label=elastic-otel-javaagent)](https://mvnrepository.com/artifact/co.elastic.otel/elastic-otel-javaagent/latest)
+
+Latest snapshot: [![Sonatype Nexus](https://img.shields.io/nexus/s/co.elastic.otel/elastic-otel-javaagent?server=https%3A%2F%2Foss.sonatype.org&label=elastic-otel-javaagent)](https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=co.elastic.otel&a=elastic-otel-javaagent&v=LATEST)
+
+## Prerequisites
+
+In order to send your data to Elastic using EDOT Java, you need to have the following prerequisites
+- an [Elastic](/quickstart/) deployment
+- an [EDOT collector](/edot-collector/) deployment
+
+TODO: how to create an API key for the OTEL SDK ? Should probably be distinct from the collector API key.
```

Comment:
[for reviewer] we need to define once for all SDKs otherwise it might imply lots of duplication. parts of the TODOs so we should fix this in this PR.